examples/z_put.py

```diff
@@ -87,7 +86,7 @@ def main():
     # - Json (str format)
     # session.put('demo/example/Json',
     #             json.dumps(['foo', {'bar': ('baz', None, 1.0, 2)}]).encode(),
-    #             encoding=Encoding.TEXT_JSON)
+    #             encoding=Encoding.TEXT_JSON())
```

Comment:
Can be `encoding=zenoh.Encoding.TEXT_JSON()` since we remove the import

Reply:
OMG, I forgot it for the second time...